s3torchconnectorclient/python/tst/unit/test_mountpoint_s3_client.py

```diff
@@ -140,19 +167,32 @@ def test_list_objects(expected_keys: Set[str]):
 
 
 @pytest.mark.parametrize(
-    "prefix, keys, expected_keys",
+    "prefix, keys, expected_keys, force_path_style",
```

Comment:
Could you please move `force_path_style` into separate `@pytest.mark.parametrize` decorator to prevent repetition for this function and for similar use cases too?
[Here](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) are example from documentation how to do that:
```
To get all combinations of multiple parametrized arguments you can stack parametrize decorators:

import pytest


@pytest.mark.parametrize("x", [0, 1])
@pytest.mark.parametrize("y", [2, 3])
def test_foo(x, y):
    pass

This will run the test with the arguments set to x=0/y=2, x=1/y=2, x=0/y=3, and x=1/y=3 exhausting parameters in the order of the decorators.
```

Reply:
Done